apps/modernization-ui/src/apps/page-builder/pages/EditPage/EditPageHeader/EditPageHeader.tsx

```diff
@@ -39,6 +39,13 @@ export const EditPageHeader = ({ page, handleSaveDraft }: PageProps) => {
                         Save draft
                     </Button>
                 )}
+                <LinkButton
+                    href={`/nbs/page-builder/api/v1/pages/${page.id}/preview`}
+                    label="open a preview of the current page"
+                    rel="noreferrer"
```

Comment:
question:  I wonder if these should be the defaults?

Reply:
I think so.